components/toast_wrapper/toast_wrapper.jsx

```diff
@@ -85,7 +85,7 @@ class ToastWrapper extends React.PureComponent {
 
         // show unread toast on mount when channel is not at bottom and unread count greater than 0
         if (typeof showUnreadToast === 'undefined' && props.atBottom !== null) {
-            showUnreadToast = unreadCount > 0 && !props.atBottom;
```

Comment:
minor nitpick, can this `1000` scroll be a named constant so it can be more easily read?

Reply:
yes done